### DISCUSSION TITLE
Store all data in LocalStorage

### DISCUSSION THREAD

**Discussion by sourabhpatole07** [mobxjs/mobx#3961] [General] [answered]
Hi...
currently we are checking all data is generated from api after login that should be possible to store in localStorage 
is that any mobx function or any method available ?
please let me know

**Comment by mweststrate** [comment]
There is not a MobX specific solution (or problem) to this, (de)serialize the data as JSON should typically do the trick. Whether that is easy or hard mostly depends on the way the data was structures.

**Comment by codeBelt** [comment, marked as answer]
You might find [mobx-persist-store](https://github.com/quarrant/mobx-persist-store) useful for your project.

**Reply by kubk** [reply]
I can also recommend this package. I’ve been using it in production for a few years, and it’s easy to customize with custom adapters or use with the localforage package.
